lib/functions/main/build-tasks.sh

```diff
@@ -6,26 +6,32 @@
 #
 # This function propagates the deprecated KERNEL_ONLY configuration
 # to the appropriate content of the BUILD_ONLY configuration.
+# If KERNEL_ONLY="yes" it actually meant to build only packages using cross
+# compilation.
+# If KERNEL_ONLY="no" it added an image assembly.
+# We are adding a collective target "default"
+#
 # It exists for backward compatibility only.
 #
 backward_compatibility_build_only() {
-	local _kernel_buildOnly="u-boot,kernel,armbian-config,armbian-zsh,plymouth-theme-armbian,armbian-firmware,armbian-bsp"
+	local _build_packages="u-boot,kernel,armbian-config,armbian-zsh,plymouth-theme-armbian,armbian-firmware,armbian-bsp"
```

Comment:
`_build_packages` is now defined in different places - see also in check.sh.
I would like to move the multiple used constants to an appropriate function echoing the constant value and then assign `_build_packages=$(CONST_build_packages)`
with:
```
CONST_build_packages() {
   echo "u-boot,kernel,armbian-config,armbian-zsh,plymouth-theme-armbian,armbian-firmware,armbian-bsp"
}
```

Reply:
backward_compatibility_build_only() This function will be removed in a month or two
when users get used to the new variable. It is for this reason that I changed the name
and allowed the presence in two places as a local variable.
This variable will only be needed in the build_validate_buildOnly() function

Reply:
I thought a little. Yes, I agree.
Especially since for "chroot" I will have to implement a separate function for analyzing directories
"${SRC}/packages/extras-buildpkgs/"
"$USERPATCES_PATH/packages/extras-buildpkgs/" to provide a list of valid names.